cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -55,6 +59,21 @@ def new_instance(provider):
         pass
 
 
+@pytest.fixture(scope='module')
```

Comment:
Q: are you sure module scope is correct ? as far as I see your instance has function scope.

Reply:
Just wanted to test something and forgot it